### PR TITLE
add entry point and fix generation of s3_credentials file

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,3 +3,5 @@ FROM nginx:latest
 LABEL org.opencontainers.image.authors="pmc@voxpupuli.org"
 
 COPY . /usr/share/nginx/html/
+
+ENTRYPOINT [ "/usr/share/nginx/html/entrypoint.sh" ]
```
Comment (ekohl) [inline Dockerfile:7]: It's a bit odd to have the entry point in the document root, but it looks like that's already a preexisting problem.

Comment (rwaffen) [reply Dockerfile:7]: i can copy this to another location...

File: Dockerfile [hunk 2]
```diff
@@ -3,3 +3,6 @@ FROM nginx:latest
 LABEL org.opencontainers.image.authors="pmc@voxpupuli.org"
 
 COPY . /usr/share/nginx/html/
```
Comment (ekohl) [inline Dockerfile:5]: Perhaps the app itself should be moved to a `html` subdirectory in the repo (or otherwise named) so you don't copy all the unrelated files too. May be out of scope for this PR.

File: Dockerfile [hunk 3]
```diff
@@ -3,3 +3,6 @@ FROM nginx:latest
...
+COPY entrypoint.sh /
```
Comment (ekohl) [inline Dockerfile:6]: I'd move this above the other `COPY`. This file is less likely to change and you can use caching more efficiently. And if this file changes, the above will also be invalidated so you'd trash the cache anyway.

File: Dockerfile (hunk 2 above)
Comment (rwaffen) [reply Dockerfile]: would also say, this would be a little bit out of scope here. but we cann do an own issue out of it

**Review by bastelfreak — APPROVED** [review]
